bin/functions/hibench_prop_env_mapping.py

```diff
@@ -62,7 +62,7 @@
     YARN_EXECUTOR_CORES="hibench.yarn.executor.cores",
     SPARK_YARN_EXECUTOR_MEMORY="spark.executor.memory",
     SPARK_YARN_DRIVER_MEMORY="spark.driver.memory",
-    DATA_HDFS="hibench.hdfs.data.dir",
```

Comment:
This configuration is used by all other workloads. The change here will break other workloads's behavior.